systers_portal/meetup/migrations/0002_meetup.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import ckeditor.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0002_auto_20150420_1504'),
+        ('meetup', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Meetup',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', serialize=False, primary_key=True)),
+                ('title', models.CharField(max_length=50, verbose_name='Title')),
+                ('slug', models.SlugField(verbose_name='Slug', unique=True)),
```

Comment:
I think it was by mistake, it should be 50, I checked in model also slug does not have any `max_length` restriction that is why migrations also not showing it. I will update model as well as migration.